content/blog/lightweight-k3s-cluster-raspberry-pi.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "k3s cluster on Raspberry Pi’s"
+date: 2022-12-27
+draft: false
+featured: true
+weight: 1
+---
+As a developer, I was tasked with setting up a local Kubernetes cluster for deploying and managing internal toolings within my company. We decided to set up the cluster on Raspberry Pis in our office. By using Raspberry Pis, we were able to utilize existing hardware and avoid expensive infrastructure costs. However, setting up a full Kubernetes cluster can be challenging, especially on resource-constrained devices like Raspberry Pis. That's why we decided to use k3s, a lightweight version of Kubernetes designed for edge computing and IoT scenarios.
```

Comment:
> local Kubernetes cluster 

Better wording.

> As a developer, I was tasked with setting up a local Kubernetes cluster for deploying and managing internal toolings within my company

Better wording. For example:

I work on managing internal toolings for Infraspec. Among other things, running office local toolings and network services falls under my ambit. We had a Raspberry PI cluster lying around. I decided to work on setting up k3s on it.